changelog/unreleased/features/2513--debian-package.md

```diff
@@ -0,0 +1,7 @@
+VAST is now distributed as a Debian Package. The package for this release can be
+obtained from
+[https://storage.googleapis.com/tenzir-public-data/vast-static-builds/vast_v2.4.0_amd64.deb].
+This package automatically installs a systemd service and creates a `vast` user
+for the database process, greatly simplifying the setup process. Refer to [the
+docs](https://vast.io/docs/setup-vast/install/linux#debian) for detailed
+instructions.
```

Comment:
Please rewrite using active voice, and link to the GitHub releases page rather than our public GCS bucket.